src/pages/commits/hooks/useCommitsLimit.ts

```diff
@@ -0,0 +1,25 @@
+import { MutableRefObject, useEffect, useRef } from "react";
+import { useDimensions } from "hooks/useDimensions";
+import { useQueryParams } from "hooks/useQueryParam";
+import { MainlineCommitQueryParams } from "types/commits";
+
+export const useCommitsLimit = <T extends HTMLElement>(): [
```

Comment:
What do you think about `useCommitLimit`?
```suggestion
export const useCommitLimit = <T extends HTMLElement>(): [
```